mempool/src/pool/reorg.rs

```diff
@@ -130,6 +130,21 @@ fn fetch_disconnected_txs<M>(
 pub fn handle_new_tip<M: MemoryUsageEstimator>(mempool: &mut Mempool<M>, new_tip: Id<Block>) {
     mempool.rolling_fee_rate.get_mut().set_block_since_last_rolling_fee_bump(true);
 
+    let is_ibd = mempool
+        .blocking_chainstate_handle()
+        .call(|cs| cs.is_initial_block_download())
+        .expect("Cannot connect to chainstate");
```

Comment:
Why not propagate the error via '?' instead of expecting?

Reply:
This is an event handler, so it does not return a value.

However, it may be improved somewhat. I think logging the error will be useful.

I am not sure if the expect should stay or be changed to early exit. There is  chance the `expect` will trigger when the node is being shut down. On the other hand, not sure just ignoring it (except for logging) is a very good idea.

Reply:
> But I'd do it on the highest level possible, which is, as I understand, the `run` method of `impl MempoolSubsystemInterface for MempoolInit`. And all functions that are called by `run` directly or indirectly IMO should just return `Result`.

Good point. Done.